src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriter.cs

```diff
@@ -914,20 +914,17 @@ protected void WriteXmlAttribute(XmlNode node, object? container)
         protected void WriteAttribute(string localName, string? ns, string? value)
         {
             if (value == null) return;
-            if (localName == "xmlns" || localName.StartsWith("xmlns:", StringComparison.Ordinal))
-            {
-                ;
-            }
-            else
+
+            if (localName != "xmlns" && !localName.StartsWith("xmlns:", StringComparison.Ordinal))
             {
                 int colon = localName.IndexOf(':');
+
                 if (colon < 0)
                 {
                     if (ns == XmlReservedNs.NsXml)
                     {
                         string? prefix = _w.LookupPrefix(ns);
-                        if (prefix == null || prefix.Length == 0)
-                            prefix = "xml";
+                        if (prefix == null || prefix.Length == 0) prefix = "xml";
```

Comment:
If we're going to change the formatting here, I'd rather we add braces rather than change it to be a one-line statement.

Reply:
Also, [`String.IsNullOrEmpty(...)`](https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorempty?view=net-5.0)

Reply:
@Clockwork-Muse not in this PR